src/main/scala/tech/cryptonomic/nautilus/cloud/adapters/doobie/ApiKeyQueries.scala

```diff
@@ -70,4 +70,9 @@ trait ApiKeyQueries extends EnvironmentMappers {
     sql"SELECT keyid, key, environment, userid, dateissued, datesuspended FROM api_keys WHERE userid = $userId AND dateissued > current_date - interval '30 days'"
       .query[ApiKey]
 
+  /** Query returning API keys connected to user active in the given range of time */
```

Comment:
Even though it would be counted in the both periods it will be inserted only once as I've added mechanism preventing adding duplicated `(userid, start, end)`. Also it is highly improbable to generate the key at the boundary .